packages/vite/src/node/plugins/css.ts

```diff
@@ -489,11 +533,9 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           }
         }
       } else {
-        // non-split extracted CSS will be minified together
-        chunkCSS = await processChunkCSS(chunkCSS, {
-          inlined: false,
-          minify: false
-        })
+        chunkCSS = resolveAssetUrlsInCss(chunkCSS, cssBundleName)
+        // finalizeCss is called for the aggregated chunk in generateBundle
+
```

Comment:
@bluwy `finalizeCss` was being called for every chunkCSS before aggregation, and then again in `generateBundle`. I removed it from here as it is better to only call it once AFAICS.